sdk/servicebus/.project

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Any idea why these show up as adds vs moves? my guess is that for some reason git didn't delete the old one, most likely becaus eof the "." in the name caused it to be filtered.

Reply:
Not sure - I actually did a clean clone to do the moves and did force ```git add . -f``` after I had made the tweaks just to make sure I got a clean move. I think it is some kind of Git quirk.

Reply:
Ok just make sure the old directory no longer exists after your change.